attributes/default.rb

```diff
@@ -221,6 +221,11 @@
 # extra_packages to install
 default['datadog']['extra_packages'] = {}
 
+# extra config options
```

Comment:
nit: could you put this right under `default['datadog']['histogram_percentiles']` in this file, so that it's visually regrouped with the other agent config attribute defaults?